test-site/.gitignore

```diff
@@ -13,6 +13,8 @@ webpack-config.js
 !config/global_config.json
 !config/locale_config.json
 !config/index.json
+!config/index.ar.json
```

Comment:
I deleted these pages to speed up the testing loop. When I added them back I noticed they were missing from the gitignore. They were probably being tracked in a similar way to `git add --force`